pages/_app.tsx

```diff
@@ -50,6 +51,32 @@ export default function App({ Component, pageProps }: AppProps): JSX.Element {
           `,
         }}
       />
+      <DefaultSeo
+        title="Mutual Aid NYC"
+        description="Our community-sourced, volunteer-curated library is a growing collection of the many resources available to New Yorkers. Mutual Aid NYC is committed to building a comprehensive list of high-quality resources—check back frequently, as new resources are added every day!"
+        canonical="https://lists.mutualaid.nyc/"
+        openGraph={{
+          type: 'website',
+          locale: 'en_US',
+          url: 'https://lists.mutualaid.nyc/',
+          siteName: 'Mutual Aid NYC',
+          title: 'Mutual Aid NYC',
+          description:
+            'Our community-sourced, volunteer-curated library is a growing collection of the many resources available to New Yorkers. Mutual Aid NYC is committed to building a comprehensive list of high-quality resources—check back frequently, as new resources are added every day!',
+          images: [
+            {
+              url: 'https://lists.mutualaid.nyc/manyc_logo.png',
```

Comment:
Can we make this a url relative to the folder this image is in the project or no?

Reply:
I don't think so? I think this is the URL OpenGraph presents to sites like Twitter so they wouldn't really be able to pass an image from a folder path? But I am a little confused on what you mean so I might have misunderstood what you asked.

Reply:
Am thinkin something like `public/manyc_logo.png` like can do with image hrefs but if it doesn't work like that nbd